common/scaffolding/interactive-content-en.html

```diff
@@ -0,0 +1,365 @@
+<circle cx="50" cy="50" r="40" stroke="red" stroke-width="5" fill="blue" />
+</svg>
+
+<pre>
+<code>&lt;svg width="100" height="100"&gt;
+&lt;circle cx="50" r="40" stroke="red" stroke-width="5" fill="blue" /&gt;
+&lt;/svg&gt;
+</code>
+</pre>
+
+<h2>The <code>details</code> and <code>summary</code> element</h2>
+
+<details>
+<summary>Summary</summary>
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus sagittis metus ligula, aliquet vulputate ligula molestie nec. Aenean dapibus varius dapibus.</p>
+</details>
+<pre>
+<code>&lt;details&gt;
+&lt;summary&gt;Summary&lt;/summary&gt;
+&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus sagittis metus ligula, aliquet vulputate ligula molestie nec. Aenean dapibus varius dapibus.&lt;/p&gt;
+&lt;/details&gt;</code>
+</pre>
+
+<h2>The <code>dialog</code> element</h2>
+
+<dialog open>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</dialog>
+<pre>
+<code>&lt;dialog open&gt;
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+&lt;/dialog&gt;</code>
+</pre>
+
+<h2>Draw graphics with the <code>canvas</code> element</h2>
+
+<canvas id="myCanvas">
+Your browser does not support the canvas tag.
+</canvas>
+
+<script>
+	let canvas = document.getElementById("myCanvas");
+	let ctx = canvas.getContext("2d");
+	ctx.fillStyle = "#FF0000";
+	ctx.fillRect(0, 0, 80, 80);
+</script>
+
+<pre>
+<code>&lt;canvas id="myCanvas"&gt;
+Your browser does not support the canvas tag.
+&lt;/canvas&gt;
+
+&lt;script&gt;
+	let canvas = document.getElementById("myCanvas");
+	let ctx = canvas.getContext("2d");
+	ctx.fillStyle = "#FF0000";
+	ctx.fillRect(0, 0, 80, 80);
+&lt;/script&gt;
+</code>
+</pre>
+
+<h2>The <code>template</code> element</h2>
+
+<p>Click the button below to display the hidden content from the template element.</p>
+
+<button onclick="showContent()">Show hidden content</button>
+
+<div id="content"></div>
+
+<template>
+<h2>Hidden Image</h2>
+<img src="img/input-image-sample.jpg" width="300" height="200">
+</template>
+
+<script>
+	function showContent() {
+		let temp = document.getElementsByTagName( "template" )[ 0 ];
+		let clone = temp.content.cloneNode( true );
+		let content = document.getElementById( "content" );
+		content.appendChild( clone );
+	}
+</script>
+
+<pre>
+<code>&lt;button onclick="showContent()"&gt;
+Show hidden content
+&lt;/button&gt;
+
+&lt;template&gt;
+	&lt;h2&gt;Hidden Image&lt;/h2&gt;
+	&lt;img src="img/input-image-sample.jpg" width="300" height="200"&gt;
+&lt;/template&gt;
+
+&lt;script&gt;
+	function showContent() {
+		let temp = document.getElementsByTagName( "template" )[ 0 ];
+		let clone = temp.content.cloneNode( true );
+		let content = document.getElementById( "content" );
+		content.appendChild( clone );
+	}
+&lt;/script&gt;
+</code>
+</pre>
+
+<h2>The <code>slot</code> element</h2>
+<button onclick="fillTemplate()">
+Fill template
+</button>
+
+<template id="slotTemplate">
+<h1><slot name="heading"></slot></h1>
```

Comment:
Change this title for an heading level 3 because we need to ensure the page keep its heading properly structured anytime in run time.

Reply:
Fixed in commit "Fixed errors and added images".